test/unit_tests/datautil/test_windowers.py

```diff
@@ -14,13 +16,15 @@
 from braindecode.datasets.moabb import fetch_data_with_moabb
 from braindecode.datautil import (
     create_windows_from_events, create_fixed_length_windows)
+from braindecode.datautil.preprocess import MNEPreproc, preprocess
```

Comment:
```suggestion
from braindecode.preprocessing import MNEPreproc, preprocess
```

would be so much nicer